### ISSUE TITLE
Keyboard navigation using cursor keys does not work in web demo

### ISSUE THREAD

**Issue by vi** [emilk/egui#512]
1. Navigate to https://emilk.github.io/egui/#demo
2. Press enough `Tab`s to get to to checkbox
3. Press Down or Up key on keyboard

Expected:

It navigates to "First" radio button or to "Click me" respectively.

Actual:

Keypress seems to be ignored.

---

Especially annoying when trying to edit combobox with keyboard only.

**Comment by akhilman** [comment]
I would also like to have navigation with an arrow keys.
But I would like to have separate events, such as focus to the left, focus on the right, focus up and focus down, so the user will be able to bind these events to anything that he/she wants, for example, Alt + Arrows, or HJKL or Gamepad's Pad.

This can be implemented as an array of tuples `(the coordinates of the widget on the screen, widget's ID)`, as well as the same touple for the current active widget.
Then we are looking for the nearest widget in the selected direction from the current widget and switch the focus to it.

**Comment by emilk** [comment]
What is needed in order to accomplish keyboard navigation has been discussed in more detail in https://github.com/emilk/egui/issues/1250, so I'll close that in favor of https://github.com/emilk/egui/issues/1250

**Comment by jabeka** [comment]
The issue (and https://github.com/emilk/egui/issues/383 which seems a duplicate of this one) is still present on egui 0.28.1 although https://github.com/emilk/egui/issues/1250  is closed.
Can a maintainer please re-open it ?
